pkg/project/v1/project_loader_test.go

```diff
@@ -25,9 +25,9 @@ import (
 	"strings"
 	"testing"
 
-	"gotest.tools/assert"
-
 	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/api"
+	"github.com/stretchr/testify/assert"
+	assert2 "gotest.tools/assert"
```

Comment:
We discussed this at some point that we want to use `testify` in all new files (and switch it slowly in old ones), but don't want to mix them.

Reply:
Yes indeed we did. And I am doing it whenever is possible...
Here is the case that one uses some wierd stuff things and is not so clear what is happen here:
```
	// NOT test-resources/hidden-directories/.logs
	// NOT test-resources/hidden-directories/project2/.logs
	assert2.DeepEqual(t, projects, []string{
		"test-resources/hidden-directories/project1",
		"test-resources/hidden-directories/project2/subproject",
	}, cmpopts.SortSlices(func(a, b string) bool { return strings.Compare(a, b) < 0 }))
```

So I changed everything else, and this one left for future....